### PR TITLE
Make sample queries more easily configurable

### PR DESCRIPTION
**Pull request by MalinAhlberg**
### Description

Allow a `sample_queries.json` file in `beacon_api/conf` with sample queries to show in the `info` call, instead of the ones in `beacon_api/api/info.py`.

Makes it easier to have your own sample queries, since you will not need to edit the `beacon_api/api/info.py` to change this.

### Type of change

- [X]  New feature (non-breaking change which adds functionality)

### Changes Made

Add another optional config-file, the content of this will be stored in `__sample_queries__`.

### CODE REVIEW

File: beacon_api/api/info.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from .. import __apiVersion__, __title__, __version__, __description__, __url__, __alturl__, __handover_beacon__
 from .. import __createtime__, __updatetime__, __org_id__, __org_name__, __org_description__
 from .. import __org_address__, __org_logoUrl__, __org_welcomeUrl__, __org_info__, __org_contactUrl__
-from .. import __handover_drs__
+from .. import  __sample_queries__,__handover_drs__
```
Comment (MalinAhlberg) [inline beacon_api/api/info.py]:
```suggestion
from .. import  __sample_queries__, __handover_drs__
```

File: beacon_api/api/info.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from .. import __apiVersion__, __title__, __version__, __description__, __url__, __alturl__, __handover_beacon__
 from .. import __createtime__, __updatetime__, __org_id__, __org_name__, __org_description__
 from .. import __org_address__, __org_logoUrl__, __org_welcomeUrl__, __org_info__, __org_contactUrl__
-from .. import __handover_drs__
+from .. import  __sample_queries__, __handover_drs__
```
Comment (MalinAhlberg) [inline beacon_api/api/info.py]:
```suggestion
from .. import __sample_queries__, __handover_drs__
```

Comment (MalinAhlberg) [reply beacon_api/api/info.py]: Remove whitespace

File: beacon_api/conf/__init__.py [hunk 3]
```diff
@@ -58,3 +59,6 @@ def parse_oauth2_config_file(path):
 
 
 OAUTH2_CONFIG = parse_oauth2_config_file(os.environ.get('CONFIG_FILE', os.path.join(os.path.dirname(__file__), 'config.ini')))
+
```
Comment (teemukataja) [inline beacon_api/conf/__init__.py]:
```suggestion
# Sample query file should be of format [{BeaconAlleleRequest}] https://github.com/ga4gh-beacon/specification/
```

Comment (teemukataja) [reply beacon_api/conf/__init__.py]: Maybe also add a comment that indicates the structure in the `sample_queries.json`?

**Comment by teemukataja** [comment]
> It would be nice in the long term if we could generate these example somehow, or start an issue and think about it.

One solution could be, that we extract a few rows from the DB as example queries (only PUBLIC datasets).

**Pull request merged**